channel/Monster3GPP38901.m

```diff
@@ -960,11 +815,12 @@ function checkInterpolationRange(axisXY, Position)
 					prop(prop~=1 & User.Position(3) <= 13) = (18./prop(prop~=1 & User.Position(3) <= 13) + exp(-1*((prop(prop~=1 & User.Position(3) <= 13))/63)).*(1-(18./prop(prop~=1 & User.Position(3) <= 13))));
```

Comment:
Holy crap. This needs to be reduced in complexity. Everything after the todo should be refactorized into a function that takes two inputs. areaType (string) and prop (scalar), height of the UE (scalar).

Reply:
And we should add references to the equation used